### ISSUE TITLE
interactsh-server LetsEncrypt DNS resolve confusion during Cert generation

### ISSUE THREAD

**Issue by mdisec** [projectdiscovery/interactsh#250]
### Interactsh version:
interactsh@v1.0.2/

### Current Behavior and Steps to Reproduce:
- I'm running my own Ubuntu server (version 20.04.4 LTS)
- Since the ubuntu runs systemd-resolved for DNS service, I stop it via  `systemctl stop systemd-resolved` 
- Run the interactsh-server via following command
```
root@mdionly:~/go/pkg/mod/github.com/projectdiscovery/interactsh@v1.0.2/cmd/interactsh-server# go run . -d rce.guru -ip 179.43.139.171

    _       __                       __       __
   (_)___  / /____  _________ ______/ /______/ /_
  / / __ \/ __/ _ \/ ___/ __ '/ ___/ __/ ___/ __ \
 / / / / / /_/  __/ /  / /_/ / /__/ /_(__  ) / / /
/_/_/ /_/\__/\___/_/   \__,_/\___/\__/____/_/ /_/ v1.0.2

                projectdiscovery.io

[INF] Requesting SSL Certificate for:  [*.rce.guru, rce.guru]
[ERR] An error occurred while applying for an certificate, error: [*.rce.guru] Obtain: [*.rce.guru] creating new order: provisioning client: performing request: Get "https://acme-v02.api.letsencrypt.org/directory": dial tcp 179.43.139.171:443: connect: connection refused (ca=https://acme-v02.api.letsencrypt.org/directory)
[ERR] Could not generate certs for auto TLS, https will be disabled
[INF] Listening with the following services:
[HTTP] Listening on TCP 0.0.0.0:80
[DNS] Listening on TCP 0.0.0.0:53
[LDAP] Listening on TCP 0.0.0.0:389
[DNS] Listening on UDP 0.0.0.0:53
[SMTP] Listening on TCP 0.0.0.0:25
```
- The problem is that Lets Encrypt certificate generation is sending HTTP request to the `https://acme-v02.api.letsencrypt.org/directory`. But the problem is interacth-server runs DNS services and returns host ip for any domain. Thus, HTTP request to the  letsencrypt.org is being redirected to the localhost. 

- I thought maybe the interactsh-server issues the cert before starting it's own DNS server. Since I only need to generate certs one time, started the `systemctl start systemd-resolved` and run the command again. In case of successfully cert generation, tool will crash  again but this time I can stop the service and run the tool one more time :) But unfortunately it does not continue since the UDP/TCP 53 ports are being used by another process.

Btw: I've tried -ip and -lip params but same error.

### Expected Behavior:
If I'm not doing anything wrong, 
Maybe DNS requests only for `letsencrypt.org` can be resolved by using external DNS service ?

**Comment by mdisec** [comment]
You know sometimes you spent so much time on a issue you have encountered.  You feel like, I cannot be the only person who encountered that in this planet, but still couldn't find a proper solution.  Eventually you decide to issue a new ticket after spending like 2 hours.

6 minutes later after you created the issue. All of a sudden, your brain kicks-in and you realize that how stupid you are. Yep, that happens. It was totally my fault.  This issue used to exist and fixed via https://github.com/projectdiscovery/interactsh/pull/11 . Yep, once again I was right. I wasn't the only one who encountered that :)
